source/module_basis/module_pw/module_fft/fft_rcom.h

```diff
@@ -6,11 +6,11 @@
 namespace ModulePW
 {
 template <typename FPTYPE>
-class FFT_ROCM : public FFT_BASE<FPTYPE>
+class FFT_RCOM : public FFT_BASE<FPTYPE>
```

Comment:
Should be rocm right 🤣

Reply:
So sorry for this problem.

Reply:
Please also fix other typos in this PR.

Reply:
Okay,thanks,it's done. Here is the current commit's DCU test status.Here are the test results using DCU for the current commit.
![img_v3_02gt_61828b1c-ef16-46eb-a224-8b0144a92acg](https://github.com/user-attachments/assets/5fbe2316-8ad8-47dc-82a1-4f46c7307a34)

Reply:
This is the result from the single version.
![img_v3_02gt_092d2ed0-5663-4ef3-9bc1-38d92c7d8feg](https://github.com/user-attachments/assets/f2af2964-a6c6-4361-b294-5a118ab2d273)

Reply:
I'm sorry but I mean `rcom` in your PR should be `rocm`, right? Please correct me if I'm wrong.

> Here is the current commit's DCU test status.Here are the test results using DCU for the current commit.

Sorry but what's the point here?

Reply:
Thank you for the reminder. I misunderstood earlier. The code has now been modified to rocm.